samples/highcharts/chart/colors-parsers/demo.details

```diff
@@ -0,0 +1,7 @@
+---
+ name: Highcharts Demo
+ authors:
+   - Torstein Hønsi
+ js_wrap: b
+ skipTest: true
```

Comment:
Remove `skipTest: true`, this sample should be part of the visual tests.

Reply:
We then need to have RGBColor part of vendors.

Reply:
You have to help me with that, I did not managed to make it work.

Reply:
It looks like I figured it out this time.